controls/postgres_spec.rb

```diff
@@ -197,7 +197,8 @@
     it { should be_file }
     it { should be_owned_by USER }
     it { should be_readable.by('owner') }
-    it { should_not be_readable.by('group') }
+    it { should be_readable.by('group') }
+    # https://github.com/geerlingguy/ansible-role-postgresql/pull/187
```

Comment:
why is this PR relevant for our hardening profile?

Reply:
There's no reason why the postgresql group shouldn't have read access to the config directory.

Reply:
yeah, I know that. my concern is, why are we putting a link to PR 187 here in the comment?

Reply:
A long time ago :) Sorry @schurzi. You're right, the link was an internal marker for me to solve the issue in the geerlingguy role. It's not relevant for the baseline itself.